deploy/rbac-cji-smoketest.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: cloud.redhat.com/v1alpha1
```

Comment:
@BlakeHolifield now that we use `bonfire deploy-iqe-cji` you don't need this file anymore, right?

Reply:
Yeah, it isn't actually called, but I wanted to leave it as a reference. Maybe that's confusing though...